Utils/Dataflow/run/data4es-start

```diff
@@ -119,10 +169,9 @@ mediator() {
   done
   flush_buffer "$1"
 }
-# ---
 
-# Create named pipes for dataflow branching
-# ---
+# Define dataflow branch (subchain)
+# Creates entry point for the branch (named pipe) and returns its name
```

Comment:
Maybe "Create", to make this line into "Do X" form like most other comments?

Reply:
Done: 56dfca5.

Reply:
Uh, actually there is also "returns" in the end of the line which should be "return". Sorry, did not see and point it out before.

Reply:
Thank you, fixed now: f02f272.